app/src/main/java/com/github/se/eventradar/ui/event/EventDetails.kt

```diff
@@ -30,210 +30,137 @@ import androidx.compose.ui.text.font.FontWeight
-                Text(
-                    text = contentCategory,
-                    color = DT_fieldContentColor,
-                    fontSize = contentTextSize
-                )
-            }
-
-            Column(
-                modifier = Modifier.weight(1f)
-            ) {
-                Text(text = headerTime, color = DT_fieldTitleColor, fontSize = contentTextSize)
-                Text(text = contentTime, color = DT_fieldContentColor, fontSize = contentTextSize)
-            }
+    Row(
+        modifier =
+            Modifier.fillMaxWidth().padding(start = widthPadding, end = widthPadding).height(64.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center) {
+          Column(modifier = Modifier.weight(1f), verticalArrangement = Arrangement.SpaceBetween) {
+            Text(text = headerCategory, color = fieldTitleColor, fontSize = contentTextSize)
+            Text(text = contentCategory, color = fieldContentColor, fontSize = contentTextSize)
+          }
+
+          Column(modifier = Modifier.weight(1f)) {
+            Text(text = headerTime, color = fieldTitleColor, fontSize = contentTextSize)
+            Text(text = contentTime, color = fieldContentColor, fontSize = contentTextSize)
+          }
         }
 
-        // register button
-        FloatingActionButton(
-            onClick = { /*TODO*/ },
-            modifier = Modifier
-                .padding(bottom = 16.dp, end = 16.dp)
-                .align(Alignment.End), // Ajustez l'espacement par le bas selon vos besoins
-            containerColor = DT_lightBackgroundColor,
-        ) {
-            Icon(
-                painter = painterResource(id = R.drawable.menu),
-                contentDescription = "register to event button",
-                modifier = Modifier.size(32.dp),
-                tint = DT_special1Color,
-            )
-
-        }
-
-        //TODO add BottomNavBar (placeholder)
-        Row(
-            modifier = Modifier
-                .background(DT_fieldContentColor)
-                .height(80.dp)
-                .fillMaxWidth()
-        ) {
-            Text(text = "add bottom nav bar")
-        }
+    // register button
+    FloatingActionButton(
+        onClick = { /*TODO*/},
+        modifier = Modifier.padding(bottom = 16.dp, end = 16.dp).align(Alignment.End),
+        containerColor = MaterialTheme.colorScheme.surfaceContainer,
+    ) {
+      Icon(
+          painter = painterResource(id = R.drawable.ticket),
+          contentDescription = "register to event button",
+          modifier = Modifier.size(32.dp),
+          tint = MaterialTheme.colorScheme.primaryContainer,
+      )
     }
 
+    BottomNavigationMenu(
+        onTabSelected = { tab -> navigationActions.navigateTo(tab) },
+        tabList = TOP_LEVEL_DESTINATIONS,
+        selectedItem = TOP_LEVEL_DESTINATIONS[0])
+  }
 }
 
-
 @Preview(showBackground = true)
 @Composable
 fun EventDetailsPreview() {
```

Comment:
nit: don't forget to remove this once the feature is implemented, Previews are generally only used during development :)

Reply:
I personally don't see a problem with keeping the preview. They can also be useful for future developers who touch this code.

Reply:
Thanks for the comment, this makes sense and I agree too, I thought that Previews could slow down the app, but since our app doesn't consume too much resources right now I think it's ok + useful for future developers.